File: alpaul_gls/Content/Downloadable/PHAO2018092520181018161350/Business/TOrganismo.vb
Imports System.Collections.Generic

Public Class TOrganismo
    Public Shared Function Add(value As Models.TOrganismo) As Integer
        Return Data.TOrganismo.Add(value)
    End Function

    Public Shared Function Edit(value As Models.TOrganismo) As Integer
        Return Data.TOrganismo.Edit(value)
    End Function

    Public Shared Function Delete(value As String) As Integer
        Return Data.TOrganismo.Delete(value)
    End Function

    Public Shared Function GetById(value As String) As Models.TOrganismo
        Return Data.TOrganismo.GetById(value)
    End Function

    Public Shared Function GetByGUID(value As String) As Models.TOrganismo
        Return Data.TOrganismo.GetByGUID(value)
    End Function

    Public Shared Function GetAll() As List(Of Models.TOrganismo)
        Return Data.TOrganismo.GetAll()
    End Function
End Class

File: alpaul_gls/Content/Downloadable/PHAO2018092520181018171302/Business/TOrganismo.cs
using System.Collections.Generic;

namespace Business
{
    public class TOrganismo
    {
	    private static Data.TOrganismo obj = new Data.TOrganismo();

        public static int Add(Models.TOrganismo value)
        {
            return obj.Add(value);
        }

        public static int Edit(Models.TOrganismo value)
        {
            return obj.Edit(value);
        }

        public static int Delete(stirng value)
        {
            return obj.Delete(value);
        }

        public static Models.TOrganismo GetById(string ID)
        {
            return obj.GetById(ID);
        }

		public static Models.TOrganismo GetByGUID(string ID)
        {
            return obj.GetByGUID(ID);
        }

        public static List<Models.TOrganismo> GetAll()
        {
            return obj.GetAll();
        }
    }
}using System.Collections.Generic;

namespace Business
{
    public class TOrganismo
    {
	    private static Data.TOrganismo obj = new Data.TOrganismo();

        public static int Add(Models.TOrganismo value)
        {
            return obj.Add(value);
        }

        public static int Edit(Models.TOrganismo value)
        {
            return obj.Edit(value);
        }

        public static int Delete(stirng value)
        {
            return obj.Delete(value);
        }

        public static Models.TOrganismo GetById(string ID)
        {
            return obj.GetById(ID);
        }

		public static Models.TOrganismo GetByGUID(string ID)
        {
            return obj.GetByGUID(ID);
        }

        public static List<Models.TOrganismo> GetAll()
        {
            return obj.GetAll();
        }
    }
}